tests/distro/test_default.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2014 Microsoft Corporation
```

Comment:
I'm not familiar with python testing much but shouldn't this file be `tests/common/osutil/test_default.py`?